.gitignore

```diff
@@ -1,3 +1,6 @@
+# Build files
+dist/
```

Comment:
I notice this is in gitignore, but also added to the repo? Seems like we need it, so probs shouldn't be here, should it?

Reply:
i think it just needs a history purge, opening a ticket